src/manipulation.js

```diff
@@ -193,14 +193,14 @@ function domManip( collection, args, callback, ignored ) {
 						!dataPriv.access( node, "globalEval" ) &&
 						jQuery.contains( doc, node ) ) {
 
-						if ( node.src ) {
+						if ( node.src && !/^module$/i.test( node.type ) ) {
```

Comment:
We don't like inlining regex definitions; this should be `node.type.toLowerCase() === "module"` unless a supported browser has non-string type properties, in which case the regex should be defined outside the function.